framework/src/modules/chain/synchronizer/fast_chain_switching_mechanism.js

```diff
@@ -197,7 +197,7 @@ class FastChainSwitchingMechanism extends BaseSynchronizer {
 		try {
 			for (const block of blocks) {
 				addBlockProperties(block);
-				await this.processor.validate(block);
+				await this.processor.validateDetached(block);
```

Comment:
I think purpose of the FCS is to use validate instead of validateDetached

Reply:
Agreed, changed it already, thanks! :)